### PR TITLE
[CCTRI-1958] & [CCTRI-1953] Add support of Configuration Token, tweaks according to Michael's notes

### PR DESCRIPTION
**Pull request by AnastasiiaRozlyvan**

### CODE REVIEW

File: src/container/Content/components/Properties/propertiesReducer.js [hunk 1]
```diff
@@ -42,6 +43,26 @@ export const propertiesReducer = (state = initialState, action) => {
...
+        case UPDATE_CONF_TOKEN:
+            return {...state,
+                [action.payload.name]: action.payload.value
+            }
+
+        case TOGGLE_ALGORITHM:
+            if (action.payload === 'RS256'){
+                return Object.fromEntries(
+                    Object.entries(state).filter(
+                        ([key, val]) => key !== 'configuration-token-key'
```
Comment (mr-pavlenko) [inline src/container/Content/components/Properties/propertiesReducer.js:55]: configuration-token-key should be in const

File: src/container/Content/components/Properties/propertiesReducer.js [hunk 2]
```diff
@@ -42,6 +43,26 @@ export const propertiesReducer = (state = initialState, action) => {
...
+                    )
+                )
+            }
+            else {
+                return Object.fromEntries(
+                    Object.entries(state).filter(
+                        ([key, val]) => key !== 'custom_jwks_host'
```
Comment (mr-pavlenko) [inline src/container/Content/components/Properties/propertiesReducer.js:62]: the same

File: src/container/Content/components/Sidebar/Sidebar.js [hunk 3]
```diff
@@ -43,7 +47,7 @@ class Sidebar extends React.Component {
     onSaveButtonClick = () => {
         if (this.isValidForm()) {
             const formattedData = this.constructValidJSON();
-            const fileData = JSON.stringify(formattedData);
+            const fileData = JSON.stringify(formattedData, null, 2);
```
Comment (mr-pavlenko) [inline src/container/Content/components/Sidebar/Sidebar.js:50]: why 2?

File: src/utils/formattingUtils.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import * as constants from 'globals/constants/constants'
+
+export function arrangeJSON(data) {
+    let arrangedData = {}
+    for (const property of constants.JSON_ELEMENTS_ORDER) {
+        if (data[property] !== undefined) {
```
Comment (mr-pavlenko) [inline src/utils/formattingUtils.js:6]: !== undefined - bad practice, rewritte

File: src/container/Content/components/Sidebar/Sidebar.js (hunk 3 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/components/Sidebar/Sidebar.js:50]: @mr-pavlenko Are you suggesting 4 would be better?

File: src/utils/formattingUtils.js (hunk 4 above)
Comment (AnastasiiaRozlyvan) [reply src/utils/formattingUtils.js]: Done, replaced with `if (property in data)`

File: src/container/Content/components/Sidebar/Sidebar.js (hunk 3 above)
Comment (AnastasiiaRozlyvan) [reply src/container/Content/components/Sidebar/Sidebar.js]: replaced with 4

**Pull request merged**